docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/introduction-to-channels.md

```diff
@@ -1,22 +1,12 @@
 # Introduction to Channels
 
-Liferay Commerce provides tools designed to meet the needs of complex B2B, multi-region, and multi-market stores and simpler B2C use cases. One way that Commerce enables a wide variety of use cases is through the _Channels_ feature.
```

Comment:
I don't find this particularly helpful information and therefore suggest removing it entirely.